app/com/linkedin/drelephant/spark/fetchers/statusapiv1/statusapiv1.scala

```diff
@@ -179,6 +179,7 @@ trait TaskData{
 trait TaskMetrics{
```

Comment:
Would it be useful to have totalCpuTime in ExecutorSummary (it would be fewer calls to get the info)? It seems good to have at the stage level, so that users know which stage is problematical.